.golangci.yml

```diff
@@ -46,14 +46,20 @@ linters-settings:
       - experimental
     disabled-checks:
       - wrapperFunc
+      - whyNoLint # requires an explanation for each nolint. requires many changes
 
 linters:
   enable-all: true
   disable:
     - maligned
     - prealloc
     - gochecknoglobals
-    - wsl # requires many changes, enable it later
+    - testpackage # requires that go test files are in a different package, but current tests are on private functions
+    - wsl # requires whitelines between blocks. requires many changes, enable it later
+    - nolintlint # requires explanation for each "nolint" directive. requires many changes
+    - gomnd # flags all "magic numbers" in the code. requires many changes
+    - goerr113 # don't allow dynamic errors, requires wrapped static errors. requires many changes
+    - nestif # Doesn't allow too long nested if. requires complex changes
```

Comment:
yes, indeed, that's a lot of disable... you should plan to start fixing them next week